package.json

```diff
@@ -38,6 +38,7 @@
   },
   "dependencies": {
     "ejs": "^2.6.1",
+    "fs-extra": "^7.0.1",
```

Comment:
Why do we need this?

Reply:
In order to be able to recursively remove directory.
Plain fs has no such feature in an easy-to-implement form.